workflows/emulation/Makefile

```diff
@@ -0,0 +1,29 @@
+SUBSAMPLE_CONFIG ?=
+TRAINING_CONFIG ?=
+SUBSET_DESTINATION ?= /tmp/subset_training_data
+
+subsample:
+	python subset_training.py ${SUBSAMPLE_CONFIG}
+
+subsample_hybrid_train: SUBSAMPLE_CONFIG = simple_hybridedmf_train.yaml
+subsample_hybrid_train: subsample
+
+subsample_hybrid_valid: SUBSAMPLE_CONFIG = simple_hybridedmf_valid.yaml
+subsample_hybrid_valid: subsample
+
+get_subsampled_training: SUBSET_SOURCE = gs://vcm-ml-scratch/andrep/all-physics-emu/training-subsets/simple-phys-hybridedmf-10day/
+get_subsampled_training:
+	mkdir -p ${SUBSET_DESTINATION}/training/
+	gsutil -m cp -r ${SUBSET_SOURCE} ${SUBSET_DESTINATION}/training/
+
+get_subsampled_testing: SUBSET_SOURCE = gs://vcm-ml-scratch/andrep/all-physics-emu/validation-subsets/simple-phys-hybridedmf-10day/
+get_subsampled_testing:
+	mkdir -p ${SUBSET_DESTINATION}/testing/
+	gsutil -m cp -r ${SUBSET_SOURCE} ${SUBSET_DESTINATION}/testing/
```

Comment:
Currently added local download targets because I didn't know a clean way to open remote netcdf files using fsspec for use in the generator.